specification/netapp/resource-manager/Microsoft.NetApp/preview/2025-07-01-preview/netapp.json

```diff
@@ -0,0 +1,12140 @@
+          {
+            "name": "body",
+            "description": "The bucket details including user details, and the volume path that should be mounted inside the bucket.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/bucket"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/PoolName"
+          },
+          {
+            "$ref": "#/parameters/VolumeName"
+          },
+          {
+            "$ref": "#/parameters/BucketName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Bucket updated",
+            "schema": {
+              "$ref": "#/definitions/bucket"
+            }
+          },
+          "201": {
+            "description": "Ok - Bucket created",
+            "headers": {
+              "Location": {
+                "description": "URL that can be used to check the status of the asynchronous operation",
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "URL that can be used to check the status of the asynchronous operation",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/bucket"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v6/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Buckets_CreateOrUpdate": {
+            "$ref": "examples/Buckets_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Buckets"
+        ],
+        "summary": "Updates a bucket for a volume",
+        "description": "Updates the details of a volume bucket.",
+        "operationId": "Buckets_Update",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "The bucket details including user details, and the volume path that should be mounted inside the bucket.",
+            "in": "body",
+            "required": false,
```

Comment:
request body is optional, is it correct?
(what update can do, if the body is not provided?)

Reply:
updated